### DISCUSSION TITLE
Avoiding 2FA?

### DISCUSSION THREAD

**Discussion by GHuserE** [community/community#123119] [New to GitHub] [answered]
I keep having this banner calling me to use 2FA. Is there a way to not needing to do this? I have already lost my phone once and its not fun but loosing access to my accounts would add to the pain.

**Comment by CopperEagle** [comment, marked as answer]
Yeah @GHuserE, few like it. There is no way around it as far as I know. But Github does provide a number of options to ensure your access.

First, there are many options how to do 2FA. Mobile is an option, but if you don't want to use it, use an Authenticator App. My recommendation is [KeePassXC](https://keepassxc.org/), which is open-source, free and runs on Windows, Mac, Linux and even as browser extension if I recall correctly.

Next, there is the recovery codes. Save it to two external memory devices, optimally. These are single use codes you can use to log in and reset the 2FA.

Finally, there is the email reset option, documented [here](https://docs.github.com/en/authentication/securing-your-account-with-two-factor-authentication-2fa/recovering-your-account-if-you-lose-your-2fa-credentials). It takes several days. Just make sure you do not loose access to your email account too and verify your email (which you should have done already). The process does take several days though.

So... there are options. But they want us to do 2FA. 

Edit: Added the link for KeePassXC. No I have no affiliation with them. It's a genuine recommendation, since it is open source and quite good.

**Reply by GHuserE** [reply]
Thanks for the link, I'll look at it.
